core/iwasm/fast-jit/fe/jit_emit_conversion.c

```diff
@@ -50,31 +50,19 @@ jit_compile_op_i32_trunc_f32(JitCompContext *cc, bool sign, bool saturating)
         sign ? NEW_CONST(F32, 2147483648.0f) : NEW_CONST(F32, 4294967296.0f);
 
     if (!saturating) {
-        /*if (value is Nan) goto exception*/
-#if defined(BUILD_TARGET_X86_64) || defined(BUILD_TARGET_AMD_64)
-        /* Set nan_ret to x86::eax */
-        nan_ret = jit_codegen_get_hreg_by_name("eax");
-#else
+        /* If value is NaN, throw exception */
         nan_ret = jit_cc_new_reg_I32(cc);
-#endif
-        insn =
-            GEN_INSN(CALLNATIVE, nan_ret, NEW_CONST(PTR, (uintptr_t)isnanf), 1);
-        if (!insn) {
+        if (!jit_emit_callnative(cc, isnanf, nan_ret, &value, 1)) {
```

Comment:
I forgot to change here. So let me know if we are going to stick on the standard c99 function or not,  and I will scan all references of math functions, and make them align to the standard.